### DISCUSSION TITLE
How to get the "original" key binding for 's' back?

### DISCUSSION THREAD

**Discussion by AndreasAZiegler** [LazyVim/LazyVim#4669] [Q&A] [answered]
I like to use the vim/neovim function of the `s` key (to replace text). LazyVim seems to map the `s` key to a plugin. How can I restore the `s` key to the original neovim function?

In `keymaps.lua` I added
```
vim.keymap.set("n", "s", "s", { noremap = true, silent = true })
vim.keymap.set("n", "S", "S", { noremap = true, silent = true })
```
without success.

**Comment by dpetka2001** [comment, marked as answer]
`cl` does the same thing as `s` default behavior.

To disable current `s` just do in file `~/.config/nvim/lua/plugins/flash.lua`
```lua
return {
  {
    "folke/flash.nvim",
    keys = {
      { "s", mode = { "n", "x", "o" }, false },
      { "some_other_key", mode = { "n", "x", "o" }, function() require("flash").jump() end, desc = "Flash" },
    },
  },
}
```
If you want the `flash.nvim` to still work but on a different keymap, then make sure to create a new mapping instead of `s`.
